### ISSUE TITLE
Can't install from pypi

### ISSUE THREAD

**Issue by cemsbr** [kytos/python-openflow#340]
To reproduce, try to install python-openflow using venv:
```
python3.6 -m venv /tmp/testpypi
source /tmp/testpypi/bin/activate
python3.6 -m pip install --upgrade python-openflow
```
Current output:
```
Collecting python-openflow
  Using cached python-openflow-2017.1b1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-5ksj2nk9/python-openflow/setup.py", line 82, in <module>
        requirements = [i.strip() for i in open("requirements.txt").readlines()]
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```

**Comment by cemsbr** [comment]
We must share how packaging is done.

**Comment by diraol** [comment]
@cemsbr, I think this was addressed at #338 (that I think is fixed by #339). But the updated version wasn't uploaded to pypi, so it will not work until the next release. Unless we do a intermediate update to pypi (`2017.1b1.post1` for example)

**Comment by cemsbr** [comment]
My approach was to remove the file from setup.py, but including it in the package will be useful if we want to add a dependency later. I'm closing this as a duplicate.
